app/map/map.html

```diff
@@ -8,7 +8,7 @@
     <title>Google Map</title>
 
     <!-- Uncomment for local development -->
-    <link rel="stylesheet" href="../static/vendor/css/app.css">
+    <link rel="stylesheet" href="../static/sass/app.scss">
```

Comment:
It's bad idea to have a link to SCSS file in your html page. You have to compile it down to CSS first. You need discard your changes here.